packages/landingpage/app/[lang]/_components/layout/header/contactButton.tsx

```diff
@@ -1,16 +1,19 @@
-import useTranslation from '../../utils/hooks/useTranslation';
 import Link from 'next/link';
-import { SubRoutes } from '../../utils/routes';
-import { InoButton } from '@elements';
+import { SubRoutes } from '@routes';
 import styles from './contactButton.module.scss';
 import React from 'react';
+import { usePathname } from 'next/navigation';
+import { InoButton } from '@inovex.de/elements-react';
+import { useTranslation } from '@hooks/useTranslation';
 
 const ContactButton = () => {
-  const { t, locale } = useTranslation();
+  const pathName = usePathname()?.split('/')[1] ?? 'en';
```

Comment:
I've seen this splitting pathname logic now many times to retrieve the current language. Is there no better way to do this? If not, can we wrap create some kind of utility (either a part of the translation hook or a wrapper for the `usePathname` hook) to do this stuff?

Reply:
I replaces this kind of path splits regarding the langugage parameter with the `useTranslation` hook 👍